### PR TITLE
Add synopsis and CSS

### PR DESCRIPTION
**Pull request by FMichel5173**

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ function App() {
       <h1>Movies</h1>
       <p>
         Filtres actuels : Par popularité et comprenant au moins l'un des genres
-        suivants : Action, Aventure, Fantastique
+        suivants :Aventure, Fantastique
```
Comment (rocambille) [inline frontend/src/App.jsx]: space after colon ;)

File: frontend/src/components/Movie.jsx [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
+      <div className="container">
+        <img
+          src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
+          alt={movie.path}
+          className="imagetohover"
+        />
+        <div className="middle">
+          <div className="texthover">{movie.overview}</div>
+        </div>
+      </div>
```
Comment (rocambille) [inline frontend/src/components/Movie.jsx]: a container for an image and related data : it's a figure, not a div

File: frontend/src/components/Movie.jsx [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
   return (
-    <section>
-      <h1>{movie.title}</h1>
-      <img
-        src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
-        alt={movie.path}
-      />
-    </section>
+    <div>
+      <h5>{movie.title}</h5>
```
Comment (rocambille) [inline frontend/src/components/Movie.jsx]: div should not have titles : section should

Comment (rocambille) [inline frontend/src/components/Movie.jsx]: h1, not h5

File: frontend/src/components/Movie.jsx [hunk 4]
```diff
@@ -1,23 +1,28 @@
...
   );
 }
-
 Movie.propTypes = {
   movie: PropTypes.element.isRequired,
   title: PropTypes.element.isRequired,
-  path: PropTypes.element.isRequired,
   backdrop_path: PropTypes.element.isRequired,
+  path: PropTypes.element.isRequired,
+  overview: PropTypes.element.isRequired,
```
Comment (rocambille) [inline frontend/src/components/Movie.jsx]: you only have 1 prop : movie. the other entries are properties of the movie object (take a look at PropTypes.shape)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/App.jsx (hunk 1 above)
Comment (FMichel5173) [reply frontend/src/App.jsx]: done

File: frontend/src/components/Movie.jsx (hunk 2 above)
Comment (FMichel5173) [reply frontend/src/components/Movie.jsx]: done

Comment (FMichel5173) [reply frontend/src/components/Movie.jsx]: done

File: frontend/src/components/Movie.jsx (hunk 3 above)
Comment (FMichel5173) [reply frontend/src/components/Movie.jsx]: done

Comment (FMichel5173) [reply frontend/src/components/Movie.jsx]: done

File: frontend/src/components/Movie.jsx (hunk 4 above)
Comment (FMichel5173) [reply frontend/src/components/Movie.jsx]: done

File: frontend/src/components/Movie.jsx [hunk 5]
```diff
@@ -1,23 +1,28 @@
...
-      />
+      <figure className="container">
+        <img
+          src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
+          alt={movie.path}
+          className="imagetohover"
+        />
+        <div className="middle">
+          <div className="texthover">{movie.overview}</div>
+        </div>
```
Comment (rocambille) [inline frontend/src/components/Movie.jsx]: so use a figcaption, not a div :p

(and the inner div looks more like a p)

File: frontend/src/components/Movie.jsx (hunk 4 above)
Comment (rocambille) [reply frontend/src/components/Movie.jsx]: and commited/pushed ?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/Movie.jsx (hunk 5 above)
Comment (FMichel5173) [reply frontend/src/components/Movie.jsx]: done

File: frontend/src/components/Movie.jsx [hunk 6]
```diff
@@ -1,23 +1,27 @@
...
   );
 }
-
 Movie.propTypes = {
   movie: PropTypes.element.isRequired,
   title: PropTypes.element.isRequired,
-  path: PropTypes.element.isRequired,
   backdrop_path: PropTypes.element.isRequired,
+  overview: PropTypes.element.isRequired,
 };
```
Comment (rocambille) [inline frontend/src/components/Movie.jsx]: I'm still waiting...

**Review by rocambille — CHANGES_REQUESTED** [review]

**Pull request merged**